### PR TITLE
Add retrigger builds flag

### PR DESCRIPTION
**Pull request by SHxKM**

### CODE REVIEW

File: src/main/resources/hudson/plugins/spotinst/cloud/SpotinstTokenConfig/config.jelly [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
         <tr>
             <td>
-                <img style="width:32px;height:32px;border-radius:3px"
-                     src="https://s3.amazonaws.com/spotinst-emails/layout/spotinst-icon.png"/>
+                <img style="height:32px;"
+                     src="https://spotinst-public.s3.amazonaws.com/assets/emails/welcome-email/spot_large.png"/>
```
Comment (adivos) [inline src/main/resources/hudson/plugins/spotinst/cloud/SpotinstTokenConfig/config.jelly:9]: why this change ?

**Review by adivos — APPROVED** [review]

File: src/main/resources/hudson/plugins/spotinst/cloud/SpotinstTokenConfig/config.jelly (hunk 1 above)
Comment (SHxKM) [reply src/main/resources/hudson/plugins/spotinst/cloud/SpotinstTokenConfig/config.jelly:9]: This is a logo update requested by product.
